docs/source/api.rst

```diff
@@ -184,6 +308,174 @@ HVAC
+.. [#] ``heatEfficiencyUnits`` choices are "AFUE" and "percent".
+.. [#] Must reference an defined ``hvacHeatingSystem.id``
+.. [#] Required when ``backupSystem.systemType`` is "separate".
+
+HVAC Air Distribution Systems
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each separate air distribution system can be entered in ``...building.systems.hvac.hvacDistributionSystems.airDistributionSystems``.
+
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+Property                           Type     Units                       Constraints     Required  Default             Notes
+=================================  =======  ==========================  ==============  ========  ==================  ==============================================       
+``id``                             id                                   Must be unique  yes       PSC    
+``systemType``                     string                               see [#]_        yes       PSC  
+``numberOfReturnRegisters``        integer  count                       >=0             no        PSC
+``conditionedFloorAreaServed``     float    ft2                         >0              no        PSC
+``ducts``                          object                                               yes
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+
+.. [#] ``systemType`` choices are "regular velocity" and "gravity".
+
+``ducts`` object uses the following schema:
+
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
+Property                           Type     Units                       Constraints     Required  Default             Notes
+=================================  =======  ==========================  ==============  ========  ==================  ==============================================       
+``id``                             id                                   Must be unique  yes       PSC
+``systemType``                     string                               see [#]_        yes   both supply and return must be defined  both supply and return must be defined
+``insulationRValue``               float    F-ft2-hr/Btu                >=0             no        0   
+``leakageValue``                   float    see ``leakageUnits``        >=0             no        BSA    
+``leakageUnits``                   string                               see [#]_        no        Percent 
+``location``                       string                               see [#]_        no        see notes   
+=================================  =======  ==========================  ==============  ========  ==================  ============================================== 
```

Comment:
This table is not displaying in the final docs ([link](https://docs-radiantlabs-co--7.org.readthedocs.build/projects/modeling-api/en/7/api.html#hvac-air-distribution-systems)). May it be caused by the incorrect padding for the `systemType` row?

Reply:
good catch. I edited - we'll see if that fixes it.